feat: add vim hjkl to move around menus

Alimedhat000 opened the pull request.

Alimedhat000 committed:
feat: add vim hjkl to move around menus

HenryLoM commented:
Hi, and thank you for the effort you put in it, appreciate it!
But could you wrap it up and make a variable so it checks settings.json? So in the json file we must set true or false for vim naviagation?
I'd love to keep code clean and customizable for everyone, thanks again!

Alimedhat000 commented:
Of course!
I’ll look into it

Alimedhat000 committed:
feat: Add configurable vim navigation setting

add a vimnavigation boolean in `settings.json`
`SetGlobalKeys` now conditionally enables vim navigation

Alimedhat000 committed:
chore: show that vim must be enabled in settings.json
